src/widgets/DataTable/DataTableRow.js

```diff
@@ -56,11 +56,14 @@ const DataTableRow = React.memo(
       editableCellText,
       editableCellTextView,
       editableCellUnfocused,
+      selectedRow,
       alternateRow,
       row,
     } = newDataTableStyles;
 
-    const rowStyle = rowIndex % 2 === 0 ? alternateRow : row;
+    const { isSelected = false } = rowState || {};
+    // If the row is selected, use selectedRow style, otherwise alternate row style on index.
+    const rowStyle = isSelected ? selectedRow : (rowIndex % 2 === 0 && alternateRow) || row;
```

Comment:
a rare case where the destructuring is unhelpful - `row` by itself didn't tell me that it was a style object, whereas `newDataTableStyles.row` would